backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceHeaderImplBuilder.java

```diff
@@ -715,4 +726,34 @@ public void setC_PaymentInstruction_ID(final int c_PaymentInstruction_ID)
 		C_PaymentInstruction_ID = c_PaymentInstruction_ID;
 	}
 
+
+	public int getC_Harvesting_Calendar_ID()
+	{
+		return C_Harvesting_Calendar_ID;
+	}
+
+	public void setC_Harvesting_Calendar_ID(final int calendarId)
+	{
+		C_Harvesting_Calendar_ID = checkOverrideID("C_Harvesting_Calendar_ID", C_Harvesting_Calendar_ID, calendarId);
```

Comment:
if the harvesting aggregation-rule is not active, then metasfresh might try to aggregate two ICs with different `C_Harvesting_Calendar_ID`s this shall **not** fail, but instead, the resulting invoice shall have a `null` `C_Harvesting_Calendar_ID`. Same for all 3 new columns

Reply:
req addressed